.github/workflows/develop.yml

```diff
@@ -115,6 +118,9 @@ jobs:
           sed -i ''s^#DB_USER#^$DB_USER^g'' ./deployment/deployment.yml
           sed -i ''s^#DB_PASS#^$DB_PASS^g'' ./deployment/deployment.yml
           sed -i ''s^#DB_NAME#^$DB_NAME^g'' ./deployment/deployment.yml
+          sed -i ''s^#DB_KEEPALIVE_IDLE#^$DB_KEEPALIVE_IDLE^g'' ./deployment/deployment.yml
```

Comment:
Don't these strings needed to be added to deployment/deployment.yml too?

And can we add them as values not secureValues, I think that means we can actually see what they are in Azure? (as we won't be able to in GitHub)

Reply:
Yes, thanks, good spot.

It's possible to see any secret value sent to an Azure container instance  by connecting to the container and echoing the value to the terminal, butI agree  it would make sense to have these as `value`s too, so this step is not needed.

It would also be good to have them in Github as variables, not secrets. What do you think? I think last time I suggested this you weren't sure because the existing non-secret values were entered as secrets.